src/views/ContactmomentView.vue

```diff
@@ -246,4 +252,12 @@ aside {
     }
   }
 }
+
+.informatie-container {
```

Comment:
I think we might not need this div. If you put display:grid on .klant-panel, with a gap of var(--spacing-large), you get the same effect. This makes the 'Klanten zoeken' button drop a bit, but I think that doesn't look bad.

Reply:
Also, unrelated to your PR: if you change `ul li article` to `ul li > article` on line 196, this fixes the margins in the customer details panel.